### PR TITLE
add price monitoring test

### PR DESCRIPTION
**Pull request by daro-maj**

### CODE REVIEW

File: tests/market/test_market.py [hunk 1]
```diff
@@ -13,9 +14,14 @@
 
 wallets = [MM_WALLET, MM_WALLET2, TERMINATE_WALLET]
 
-row_selector = '[data-testid="tab-all-markets"] .ag-center-cols-container .ag-row'
+table_row_selector = '[data-testid="tab-all-markets"] .ag-center-cols-container .ag-row'
```
Comment (mattrussell36) [inline tests/market/test_market.py:17]: This variable name is misleading. The selector is not a table row but the container element of the whole table. Would it not be better named something like `all_markets_tab`?

File: tests/market/test_market.py [hunk 2]
```diff
@@ -123,23 +129,29 @@ def test_open_market(vega, page):
         is_amendment=False,
     )
 
-    submit_order(vega, MM_WALLET, market_id, "SIDE_BUY",
+    expect(page.get_by_test_id(liquidity_supplied
+                               ).get_by_test_id(item_value)).to_have_text("0.00 (0.00%)")
+    submit_order(vega, MM_WALLET.name, market_id, "SIDE_BUY",
```
Comment (mattrussell36) [inline tests/market/test_market.py:134]: Perhaps add some comments here to explain how/what you are doing to the market

**Review by mattrussell36 — CHANGES_REQUESTED** [review]

File: tests/market/test_market.py (hunk 1 above)
Comment (daro-maj) [reply tests/market/test_market.py:17]: hymm, but it is  the table row selector in all markets tab:
image

File: tests/market/test_market.py (hunk 2 above)
Comment (daro-maj) [reply tests/market/test_market.py:134]: done

File: tests/market/test_market.py (hunk 1 above)
Comment (mattrussell36) [reply tests/market/test_market.py:16]: Think I only read the firs part of the selector.

**Review by mattrussell36 — APPROVED** [review]

**Pull request merged**